activejob/lib/active_job/queue_adapters/sidekiq_adapter.rb

```diff
@@ -18,6 +18,13 @@ module QueueAdapters
     #
     #   Rails.application.config.active_job.queue_adapter = :sidekiq
     class SidekiqAdapter < AbstractAdapter
+      def check_adapter
+        ActiveJob.deprecator.warn <<~MSG.squish
+          The `sidekiq` adapter is deprecated and will be removed in Rails 8.1.
+          Please upgrade `sidekiq` gem to version 8.0 or later.
```

Comment:
```suggestion
          The built-in `sidekiq` adapter is deprecated and will be removed in Rails 8.1.
          Please upgrade `sidekiq` gem to version 8.0 or later to use the `sidekiq` gem's adapter.
```

Same thing here, feel free to tweak my suggestion further.